### PR TITLE
DIsplayHtmlWithImages: added the dynamic section script

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CRTX-81822

## Description
Added a new dynamic-section script in order to display HTML with image reference as `cid:image_id`

## Screenshots
image

## Minimum version of Cortex XSOAR
- [x] 6.5.0

### CODE REVIEW

**Review by DeanArbel — APPROVED** [review]
Looks good

File: Packs/CommonScripts/Scripts/DisplayHTMLWithImages/DisplayHTMLWithImages.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+
+def main(args):
+    incident = demisto.incident()
+    custom_fields = incident.get('CustomFields', {})
+    email_from = custom_fields.get('emailfrom')
+    email_cc = custom_fields.get('emailcc')
+    email_to = custom_fields.get('emailto')
+    email_subject = custom_fields.get('emailsubject')
+    email_html = custom_fields.get('renderedhtmll', '') or \
```
Comment (idovandijk) [inline Packs/CommonScripts/Scripts/DisplayHTMLWithImages/DisplayHTMLWithImages.py:84]: Please fix `renderedhtmll` to be `renderedhtml`

**Review by idovandijk — CHANGES_REQUESTED** [review]
Please see comment for now. I'm running a few more tests

**Review by idovandijk — CHANGES_REQUESTED** [review]
Please note that if all fields are empty it's showing some data. It should be empty instead:
image

Also when Rendered HTML was empty and Email HTML had data, the dynamic section showed all the "None" values and only then displayed the data that it was supposed to display:
image

Other than this it looked fine in my tests, great job

**Review by idovandijk — APPROVED** [review]
Great!

**Pull request merged**